components/KickoffRequestView/KickoffBody.tsx

```diff
@@ -0,0 +1,14 @@
+import { ScrollArea, Center, Text } from '@mantine/core';
+import { Prism } from '@mantine/prism';
+
+export default function KickoffBody({ body }: { body: object | undefined }) {
```

Comment:
More of a stylistic/typescript comment, but 2 things:

1. We shouldn't use `object` as a typescript type. We know this is a `fhir4.Parameters` right?
2. I'll let you choose whatever you prefer, but a different way to define props for a react component looks as follows, and I think it can be a little cleaner for viewing the props rather than inline in the function arguments, but I don't care too much either way. Regardless, for this point i think we should use `body?: type` rather than `body: type | undefined`:

``` typescript
interface Props {
  body?: fhir4.Parameters;
}

export default function KickoffBody({ body }: Props) {
```

Thoughts?

Reply:
SGTM. Implemented this in e747d96.